src/Microsoft.ML.StandardLearners/StandardLearnersCatalog.cs

```diff
@@ -266,21 +305,33 @@ public IClassificationLoss CreateComponent(IHostEnvironment env)
         /// <param name="optimizationTolerance">Threshold for optimizer convergence.</param>
         /// <param name="memorySize">Memory size for <see cref="Microsoft.ML.Learners.LogisticRegression"/>. Low=faster, less accurate.</param>
         /// <param name="enforceNoNegativity">Enforce non-negative weights.</param>
-        /// <param name="advancedSettings">A delegate to apply all the advanced arguments to the algorithm.</param>
         public static PoissonRegression PoissonRegression(this RegressionContext.RegressionTrainers ctx,
             string labelColumn = DefaultColumnNames.Label,
             string featureColumn = DefaultColumnNames.Features,
             string weights = null,
-            float l1Weight = LRArguments.Defaults.L1Weight,
-            float l2Weight = LRArguments.Defaults.L2Weight,
-            float optimizationTolerance = LRArguments.Defaults.OptTol,
-            int memorySize = LRArguments.Defaults.MemorySize,
-            bool enforceNoNegativity = LRArguments.Defaults.EnforceNonNegativity,
-            Action<PoissonRegression.Arguments> advancedSettings = null)
+            float l1Weight = LROptions.Defaults.L1Weight,
+            float l2Weight = LROptions.Defaults.L2Weight,
+            float optimizationTolerance = LROptions.Defaults.OptTol,
+            int memorySize = LROptions.Defaults.MemorySize,
+            bool enforceNoNegativity = LROptions.Defaults.EnforceNonNegativity)
         {
             Contracts.CheckValue(ctx, nameof(ctx));
             var env = CatalogUtils.GetEnvironment(ctx);
-            return new PoissonRegression(env, labelColumn, featureColumn, weights, l1Weight, l2Weight, optimizationTolerance, memorySize, enforceNoNegativity, advancedSettings);
+            return new PoissonRegression(env, labelColumn, featureColumn, weights, l1Weight, l2Weight, optimizationTolerance, memorySize, enforceNoNegativity);
```

Comment:
>PoissonRegression [](start = 23, length = 17)

does it make sense to call the other ctor, the new PoissonRegression(env, options); and just delete this ctor on this PR, or you'll tackle it later?